docs/app.md

```diff
@@ -14,6 +14,7 @@
   * [App's `preventDestroy`](#apps-preventdestroy)
   * [App's `startWithParent`](#apps-startwithparent)
   * [App's `stopWithParent`](#apps-stopwithparent)
+  * [App's `restartWithParent`](#apps-restartwithparent)
```

Comment:
nit pick but can we move this to be ordered like the app methods below?  start -> restart -> stop